client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -171,8 +171,8 @@ export default React.createClass( {
 						<Button compact primary onClick={ this.props.updateAllPlugins } >
 							{
 								this.translate(
-									'Update %(numUpdates)d Plugin',
-									'Update %(numUpdates)d Plugins',
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Update %(numUpdates)d Plugin', 'Update %(numUpdates)d Plugins' )` *ES Score: 27*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/tests/localci-test-branch)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).